app/src/main/java/org/simple/clinic/summary/PatientSummaryEffectHandler.kt

```diff
@@ -43,9 +43,19 @@ class PatientSummaryEffectHandler @AssistedInject constructor(
             uiWorkScheduler = schedulersProvider.ui()
         ))
         .addTransformer(LoadCurrentFacility::class.java, loadCurrentFacility(schedulersProvider.io()))
+        .addTransformer(HandleEditClick::class.java, handleEditClick(uiWorkScheduler = schedulersProvider.ui()))
```

Comment:
Can we add a test in `PatientSummaryEffectHandlerTest` to capture this behaviour, please? Currently, there's no regression test for this since this click was handled entirely on the UI earlier.